components/Typography/src/heading.scss

```diff
@@ -81,3 +81,9 @@
     --utrecht-heading-5-margin-block-start: var(--denhaag-space-block-2xl);
   }
 }
+
+/* stylelint-disable */
+.page-container [class^="utrecht-heading"] + *:not(.denhaag-button),
```

Comment:
This is still to global,  i applied this style locally and still blocks like the authentication block breaks.